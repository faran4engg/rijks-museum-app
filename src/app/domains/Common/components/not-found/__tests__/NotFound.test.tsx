import { create } from 'react-test-renderer';

import NotFound from '../NotFound';

jest.mock('react-router-dom', () => ({
  Link: () => '<Link />',
}));

describe('NotFound', () => {
  it('should render', () => {
    const component = create(<NotFound msg="Test Message" />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
