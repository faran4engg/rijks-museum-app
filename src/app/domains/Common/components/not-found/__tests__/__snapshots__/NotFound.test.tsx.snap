// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render 1`] = `
<div
  className="flex flex-col items-center"
>
  <svg
    aria-hidden="true"
    className="w-16 h-16 mt-8 text-theme-color-light dark:text-theme-color-dark"
  >
    error-404.svg
  </svg>
  <h1
    className="text-6xl font-semibold text-gray-700 dark:text-gray-200"
  >
    404
  </h1>
  <p
    className="my-4 text-gray-700 dark:text-gray-300"
  >
    Test Message
  </p>
  &lt;Link /&gt;
</div>
`;
